tests/PySys/environments/environment_c8y.py

```diff
@@ -0,0 +1,117 @@
+import pysys
+from pysys.basetest import BaseTest
+
+"""
+Environment to manage automated connect and disconnect to c8y
+
+Tests that derive from class EnvironmentC8y use automated connect and
+disconnect to Cumulocity. Additional checks are made for the status of
+service mosquitto and service tedge-mapper.
+"""
+
+
+class EnvironmentC8y(BaseTest):
+    def setup(self):
+
+        self.tedge = "/usr/bin/tedge"
+        self.sudo = "/usr/bin/sudo"
+
+        self.log.info("EnvironmentC8y Setup")
+        self.addCleanupFunction(self.mycleanup)
+
+        # Check if tedge-mapper is in active
+        serv_mapper = self.startProcess(
+            command="/usr/sbin/service",
```

Comment:
Hmm, for no particular reason, I was used to it. I just found out that service can also deal with "old" init scripts. It could be an advantage when we also like to support non systemd systems. However, to switch systemctl would be a clearer approach. I will think about that...

Reply:
I switched to systemctl now, wasn't a lot effort